### ISSUE TITLE
Change default alerting thresholds?

### ISSUE THREAD

**Issue by dgiffin** [dalibo/temboard#421]
Is there a way to change the default alerting thresholds when you add an instance into temboard? In an ideal world we would pre-set our standard defauls values and have them automagically picked up, at the moment we have to manually update each new instance.

**Comment by julmon** [comment]
Nope, there is no way to do that. But, this is an interesting feature request.
